tasks/aws.rake

```diff
@@ -153,7 +174,8 @@ Module.new do
         user_data: Base64.encode64(script),
         instance_type: @instance_type,
         iam_instance_profile: @web_iam_instance_profile,
-        associate_public_ip_address: true
+        associate_public_ip_address: true,
+        ipv_6_address_count: 1
```

Comment:
This doesn't actually work, launch configurations don't yet support IPv6 addresses.